video-calls/final/index.html

```diff
@@ -12,8 +12,8 @@
         <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css">
 
         <!-- Dolby.io Web SDK -->
-        <script type="text/javascript" src="https://unpkg.com/@voxeet/voxeet-web-sdk"></script>
-        <script type="text/javascript" src="../util/dolbyio-auth-helper.js"></script>
+        <script type="text/javascript" src="https://unpkg.com/@voxeet/voxeet-web-sdk/dist/voxeet-sdk.js"></script>
```

Comment:
Would you consider using this URL instead?
```
https://cdn.jsdelivr.net/npm/@voxeet/voxeet-web-sdk/dist/voxeet-sdk.js
```

Reply:
Yes, I made that change and will also update docs to change the recommended CDN.